### PR TITLE
Satan Board "Admiral Strokers" keymap and config

### PR DESCRIPTION
**Pull request by Swoy**
Added a new keymap for a satan board (60%) whit a few new layers and Space Cadett and Admiral Strokers (CTRL tap for [ and ] )

### CODE REVIEW

**Comment by Swoy** [comment]
Unsure how to resolve the fail check. Please advice.

File: keyboards/satan/keymaps/admiralStrokers/Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+MIDI_ENABLE 			=		no   # MIDI controls
+AUDIO_ENABLE 			=		no   # Audio output on port C6
+UNICODEMAP_ENABLE	=		no	 # This allows sending unicode symbols using X(<unicode>) in your keymap.
+UNICODE_ENABLE 		=		no   # Unicode
+UCIS_ENABLE 			=		no	 # Keep in mind that not all will work (See WinCompose for details on Windows).
+BLUETOOTH_ENABLE 	=		no   # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE 	=		no   # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE	=		no   # Breathing sleep LED during USB suspend
+
+VARIABLE_TRACE		=		no	 # Use this to debug changes to variable values
```
Comment (jackhumbert) [inline keyboards/satan/keymaps/admiralStrokers/Makefile]: Can you try removing this line or the tabs? The tabs here might be messing this up, causing it to turn "on."

**Comment by jackhumbert** [comment]
Have you had a chance to debug this at all?

**Comment by Swoy** [comment]
Hello, I'm in the middle of my thesis and I overlooked this for quite some time now. Thanks for the reminder. Should be sorted out now (hopefully)

**Comment by Swoy** [comment]
Is there some of my functions? I've fixed the tabs now, and updated the fork to latest.

**Comment by jackhumbert** [comment]
For some reason, it thinks variable trace is enabled on your keymap, and that's messing things up a bit. It compiles locally for you, right?

File: keyboards/satan/keymaps/admiralStrokers/keymap.c [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+				XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX,\
+				XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX,\
+			    XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX,\
+				XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, XXXX, DEAD,\
+				XXXX, XXXX, XXXX, 					XXXX,			  		XXXX, XXXX, XXXX, XXXX),
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) { // MACRODOWN only works in this function
+	static uint16_t key_timer;
+	bool checkTime(){
```
Comment (jackhumbert) [inline keyboards/satan/keymaps/admiralStrokers/keymap.c]: I'm no C expert, but maybe this needs to be defined outside the function?

File: keyboards/satan/keymaps/admiralStrokers/keymap.c [hunk 3]
```diff
@@ -0,0 +1,194 @@
+#include "satan.h"
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```
Comment (jackhumbert) [inline keyboards/satan/keymaps/admiralStrokers/keymap.c]: This line should probably be moved down to where your keymap starts - I don't think this is messing anything up though.

**Comment by jackhumbert** [comment]
Yeah, you could try that too - others that have that omitted seem to be doing alright.

File: keyboards/satan/keymaps/admiralStrokers/keymap.c (hunk 3 above)
Comment (Swoy) [inline keyboards/satan/keymaps/admiralStrokers/keymap.c]: Ok, I've looked into it. I'm totally new to this too :)

**Comment by Swoy** [comment]
I think I know where the issue lies. Hang on

**Comment by Swoy** [comment]
I'm sorry, I'm not experienced enough to figure this one out.

**Comment by fredizzimo** [comment]
Since I made the variable trace thing, I will check out what's going on there. It should of course be possible to specify it but set it to no. 

But anyway that's not the current problem you are having anymore, now it's not finding `checktime`, there's a small typo there, you declare `checkTime`, but calling `checktime`, note the uppercase `T`

File: keyboards/satan/keymaps/admiralStrokers/keymap.c [hunk 4]
```diff
@@ -147,7 +149,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 				XXXX, XXXX, XXXX, 					XXXX,			  		XXXX, XXXX, XXXX, XXXX),
 };
 
-const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) { // MACRODOWN only works in this function
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {3 // MACRODOWN only works in this function
```
Comment (fredizzimo) [inline keyboards/satan/keymaps/admiralStrokers/keymap.c]: Seems like there's an extra `3` here added by mistake. Did you try to compile it locally before pushing?

**Comment by Swoy** [comment]
and voila! Thank you for helping out! I couldn't have spotted that in a million years, considering the study state I'm in..

File: keyboards/satan/keymaps/admiralStrokers/keymap.c (hunk 4 above)
Comment (Swoy) [inline keyboards/satan/keymaps/admiralStrokers/keymap.c]: That is the weird thing here, I have the last version compiled here. And the keyboard was working as expected, even with the capitalized "T" in checktime? I have no idea how that worked out.. I'm typing on the keyboard with the keymap I initially pushed.

Is there a way to run travis prior to syncing?

**Comment by jackhumbert** [comment]
Thanks guys :)

**Pull request merged**